contribute/developer-guide.md

```diff
@@ -276,6 +276,40 @@ GRAFANA_URL=http://localhost:3000 yarn e2e:playwright
 
 Note this will not start a development server, so you must ensure that Grafana is running and accessible at the specified URL.
 
+Playwright has several commands commonly used:
+
+1 - **To open Playwright in UI**. It boosts the Grafana server and then Playwright, which runs against this server.
+
+```
+yarn e2e:playwright --ui
+```
+
+2 - **To run an individual test**. It will run the test that matches the string passed to _grep_. Playwright will run all of them if you use a string that matches multiple tests.
+
+```
+yarn e2e:playwright --grep <testname>
+```
+
+3 - **To run a project**. It will run the entire project, also known as '_suite_'. You can find them in [grafana/playwright.config.ts](https://github.com/grafana/grafana/blob/main/playwright.config.ts#L90).
+
+```
+yarn e2e:playwright --project <projectname>
+```
+
+4 - **To run tests with a specific tagname**. The tagnames are specified in each test, like you can see here in [lokiEditor.spec.ts](https://github.com/grafana/grafana/blob/main/e2e-playwright/plugin-e2e/plugin-e2e-api-tests/as-admin-user/lokiEditor.spec.ts#L7)
+
+```
+yarn e2e:playwright --grep @<tagname>
+```
```

Comment:
think we could just mention this as part of 2). `you can also run all the tests matching a specific tag with @tagName`